### ISSUE TITLE
Validation loss & accuracy never change

### ISSUE THREAD

**Issue by l-hendriks** [tflearn/tflearn#388]
I have a small dummy set of 800 samples. I use a custom metric:

```
# target_column 2 has values between 0 and 1
data, targets = load_csv('data.csv',
                        target_column = 2,
                        columns_to_ignore=[0,1],
                        categorical_labels=False,
                        n_classes=2)

# ...

def calc_distance(predictions, targets, inputs):
    return tf.reduce_sum(tf.square(predictions - targets))

# ...

net = tflearn.regression(net, loss='mean_square', metric=calc_distance)
```

To predict one continuous value from 3600 input features per sample (numbers). The network trains, but the validation acc and loss remain constant. Output:

```
Training samples: 239
Validation samples: 560
--
Training Step: 1
| Adam | epoch: 001 | loss: 0.00000 - Sum: 0.0000 | val_loss: 0.50179 - val_acc: 281.0000 -- iter: 239/239
--
Training Step: 2  | total loss: 114.27066
| Adam | epoch: 002 | loss: 114.27066 - Sum: 27310.6855 | val_loss: 0.50179 - val_acc: 281.0000 -- iter: 239/239
--
Training Step: 3  | total loss: 21.18387
| Adam | epoch: 003 | loss: 21.18387 - Sum: 5062.9434 | val_loss: 0.50179 - val_acc: 281.0000 -- iter: 239/239
--
Training Step: 4  | total loss: 5.66940
| Adam | epoch: 004 | loss: 5.66940 - Sum: 1354.9858 | val_loss: 0.50179 - val_acc: 281.0000 -- iter: 239/239
--
Training Step: 5  | total loss: 2.08914
| Adam | epoch: 005 | loss: 2.08914 - Sum: 499.3033 | val_loss: 0.50179 - val_acc: 281.0000 -- iter: 239/239
--
Training Step: 6  | total loss: 1.06620
| Adam | epoch: 006 | loss: 1.06620 - Sum: 254.8226 | val_loss: 0.50179 - val_acc: 281.0000 -- iter: 239/239
--
Training Step: 7  | total loss: 0.72523
| Adam | epoch: 007 | loss: 0.72523 - Sum: 173.3290 | val_loss: 0.50179 - val_acc: 281.0000 -- iter: 239/239
--
Training Step: 8  | total loss: 0.59736
| Adam | epoch: 008 | loss: 0.59736 - Sum: 142.7690 | val_loss: 0.50179 - val_acc: 281.0000 -- iter: 239/239
--
Training Step: 9  | total loss: 0.54471
| Adam | epoch: 009 | loss: 0.54471 - Sum: 130.1854 | val_loss: 0.50179 - val_acc: 281.0000 -- iter: 239/239
--
Training Step: 10  | total loss: 0.52131
| Adam | epoch: 010 | loss: 0.52131 - Sum: 124.5927 | val_loss: 0.50179 - val_acc: 281.0000 -- iter: 239/239
--
```

Any idea why? Each run the val_loss and val_acc are a bit different from another run, but they remain the same through all epochs? I set the validation_set quite high to make sure it's not trying only a few samples. Is it predicting exactly the same values each time?

Second, unrelated question: once this is working, I want to train this on a dataset that does not fit in RAM (it's over a TB). Is there a way to do this with tflearn?

**Comment by l-hendriks** [comment]
Oh my.. I forgot to divide pixel values with 255..  Never mind this!
